helper/logging/logger.go

```diff
@@ -121,7 +138,9 @@ func Setup(config *LogConfig, w io.Writer) (log.InterceptLogger, error) {
 		if err != nil {
 			return nil, err
 		}
-
+		if fileName == "" {
+			fileName = fmt.Sprintf("%s.log", config.DefaultFileName)
+		}
```

Comment:
This is so we default to the names supplied by config (`vault`, `agent`, `proxy` at present).